internal/scorecard/plugins/plugin_runner.go

```diff
@@ -346,3 +348,47 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 	}
 	return output, nil
 }
+
+func ListInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig) (scapiv1alpha1.ScorecardOutput, error) {
+	var suites []schelpers.TestSuite
+
+	if pluginType == BasicOperator {
+		conf := BasicTestConfig{}
+		basicTests := NewBasicTestSuite(conf)
```

Comment:
WDYT about add the logic in the new of object? Who knows more about the object than itself ;-)?